docs/guide/layout-template.md

```diff
@@ -0,0 +1,74 @@
+# Layout Templates
+
+A layout template is common in most Apostrophe apps. As the name suggests, it **contains the markup that surrounds page content and is mostly consistent across the website**. Website navigation and footers are both usually in the layout template, whether directly as markup or included from template partials.
```

Comment:
we switch between app/website. probably fine tho.

Reply:
Yeah, I'm not sure what to do with that. Don't want to box us in, but saying "app" all the time sounds pretentious.